addons/website_event_questions/static/src/js/many2many_tags_answers.js

```diff
@@ -0,0 +1,35 @@
+odoo.define('website_event_questions.many2many_tags_answers', function (require) {
+    "use strict";
+
+    var FieldMany2ManyTags = require('web.relational_fields').FieldMany2ManyTags;
+    var registry = require('web.field_registry');
+
+    var FieldMany2ManyTagsAnswers = FieldMany2ManyTags.extend({
+        className: "o_field_many2manytags o_field_many2manytags_answers",
+        /**
+         * Force the display_name to the name in order to render the name in all cases.
+         * As we create questions and answers, the display_name is computed when the 
+         * changes are saved. So to update the changes of the user, we need to set display_name 
+         * to the name because perhaps the display_name doesn't exist yet.
+         * 
+         * @override
+         */
+        _getRenderTagsContext: function () {
```

Comment:
You can remove the custom widget.

I will handle it in my task about "free text questions".